Remove mentions of filepaths that are incorrect for the debian package of
Mercury.

--- a/README
+++ b/README
@@ -22,8 +22,8 @@
 
 The documentation sources are in the `doc' subdirectory.
 The installation process will install INFO, HTML, and DVI versions under the
-nominated installation prefix, by default /usr/local/mercury-20.01.1.
-It will also install Unix-style man pages under the same prefix.
+nominated installation prefix.  It will also install Unix-style man pages
+under the same prefix.
 
 The files RELEASE_NOTES, NEWS, HISTORY, LIMITATIONS, and BUGS
 also contain useful information.  If you're interested in looking
